tests/framework/microvm.py

```diff
@@ -782,10 +782,12 @@ def ssh_network_config(
         """
         # Create tap before configuring interface.
         tapname = tapname or (self.id[:8] + "tap" + iface_id)
-        (host_ip, guest_ip) = network_config.get_next_available_ips(2)
-        tap = self.create_tap_and_ssh_config(
-            host_ip, guest_ip, network_config.get_netmask_len(), tapname
-        )
+        # The guest is hardcoded to expect an IP in a /30 network,
+        # so we request the IPs to be aligned on a /30 network
+        # See file:../../resources/tests/fcnet-setup.sh#L21
```

Comment:
Nit: Maybe worth linking to the line https://github.com/firecracker-microvm/firecracker/blob/main/resources/tests/fcnet-setup.sh#L21?

Reply:
Done